src/Reflection/BetterReflection/BetterReflectionProvider.php

```diff
@@ -201,7 +202,6 @@ public function getAnonymousClassReflection(Node\Stmt\Class_ $classNode, Scope $
 			$scopeFile,
 		);
 		$classNode->name = new Node\Identifier($className);
-		$classNode->setAttribute('anonymousClass', true);
```

Comment:
That's a good idea for a follow up PR! Although rules should better rely on `InClassNode` and `ClassReflection` so that they don't have to make any differences between traditional and anonymous classes, in theory someone could ask about `Class_::isAnonymous()` and it's always better to give the right answer :) Please take a note and send a follow up PR :)